src/views/news/NewsView/NewsView.vue

```diff
@@ -65,12 +65,12 @@
     },
   })
   export default class NewsView extends Mixins(PageDataMixin(fetchPageData)) {
-    get news() {
-      return this.pageData;
+    get news(): NewsItem {
+      return this.pageData as NewsItem;
```

Comment:
`return this.pageData!` makes it much clearer that it's just a null-strip, and not a type conversion.
I feel like this is a bad practice, but I'm not sure how it would make sense to change it.
I think a more fundamental change in how we do the `v-if`s on the pages is required for that.

Reply:
removed